README.md

```diff
@@ -138,6 +138,47 @@ Requirements:
 > Note: this project includes a [Maven Wrapper](https://maven.apache.org/wrapper/) that can be used instead of a system installed Maven.
 > Use it by replacing `mvn` by `./mvnw` on Linux and `mvnw.cmd` on Windows in the commands below.
 
+### Install JDK on M1
+In the current version, Pulsar uses a BookKeeper version which in turn uses RocksDB. RocksDB is compiled to work on x86 architecture and not ARM. Therefore, Pulsar can only work with x86 JDK. This is planned to be fixed in future versions of Pulsar.
```

Comment:
This doesn't stand any more on the nightly version. See also https://github.com/apache/pulsar/issues/12166#issuecomment-1237601981.

I agree that there can be a few troubleshooting to work on a different environment with older version, and I'd prefer a discussion thread or a mac specific section like [Hadoop does](https://github.com/apache/hadoop/blob/a73c4804d882d9bc1a424080f20c4902cb9db832/BUILDING.txt#L389-L418).

Reply:
And I just notice that it misleads me because it's not about "Install JDK on M1" - it's about "Install X86 JDK on M1".

https://www.google.com/search?q=Install+x86+JDK+on+M1 gives you:

* https://itnext.io/how-to-install-x86-and-arm-jdks-on-the-mac-m1-apple-silicon-using-sdkman-872a5adc050d
* https://github.com/sdkman/sdkman-cli/issues/1068

That can help.

Reply:
So the meaningful information here is, that if you fail to run Pulsar on M1 due to RocksDB JNI error (with a typical stack trace), the reason is that you need an X86 JDK for older versions. Then mention some methods to achieve it as you like.

Reply:
And perhaps after we release 2.11.0 and users start to use the new version, it will be less and less useful.

> M1 is notorious to have many quirks. Anything we can do to help will be greatly appreciated.

I agree. So +0 on this patch now. FYI - I encounter several other issues, and I prefer to resolve troubleshootings in discussions instead of a manual try to cover most of them:

* https://github.com/apache/pulsar/issues/16639
* https://github.com/apache/pulsar/issues/16652